engines/python/setup/djl_python/huggingface.py

```diff
@@ -173,8 +173,6 @@ def initialize(self, properties: dict):
                 self.device = int(os.getenv("LOCAL_RANK", 0))
             _rolling_batch_cls = get_rolling_batch_class_from_str(
                 self.rolling_batch_type, is_mpi, self.model_config)
-
-            # TODO: Allow user to set output formatter
             self.rolling_batch = _rolling_batch_cls(model_id_or_path,
```

Comment:
you can make output formatter as part of the kwargs, user can pass in a function if needed

Reply:
Changed.